### ISSUE TITLE
Vassal Glitch

### ISSUE THREAD

**Issue by jdoug312** [LoneGazebo/Community-Patch-DLL#1870]
Bug Report Template

Mod Version (i.e Date - (4/23b)):
5/21

1. Mod List (if using standard CPP set, leave blank):

Full CPP + Events, R.E.D. Modpack, and modmod to remove era requirement for vassalage

2. Type of error (i.e. crash, interface bug, AI quirk): 

AI quirk

3. Steps to reproduce:

When vassalage is able to be traded, put anything on your side (+2 gpt, gems, etc) first, before putting vassalage on AI's side of trade window. AI will always accept, even if it shows as impossible.

4. Additional information:

If you put vassalage on AI's side first, the AI will value it properly.
---------------------------
Supporting information:

Please note that you can attach .zip files to Issues by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files: Database.log and Lua.log needed
2. Minidump file (located in your Civ5.exe directory)
3. Screenshots (if needed)

**Comment by LoneGazebo** [comment]
@mwhiter this is something we need to fix, as vassalage is a pretty powerful game-changer.

**Comment by mwhiter** [comment]
You do not have to wait for me to get around to fixing this if it is of critical importance and you want to jump on it. I agree, it appears to be bad bug, if it can be reproduced.

**Comment by jdoug312** [comment]
Just wanted to update:

After further time spent with the latest version, I've noticed that the problem doesn't just manifest with vassalage. I was also able to get Rome to DOW Genghis (who is stronger in literally every way) by putting an offer of 2gpt on my side of the deal table before asking Caesar to do so. However, when putting the DOW Genghis on Caesar's side of the table first, he was unwilling to do it, no matter what I offered.
